src/garage/tf/models/cnn_mlp_merge_model.py

```diff
@@ -0,0 +1,162 @@
+"""CNN and MLP Merge Model."""
+import tensorflow as tf
+
+from garage.tf.models.base import Model
+from garage.tf.models.cnn_model import CNNModel
+from garage.tf.models.cnn_model_max_pooling import CNNModelWithMaxPooling
+from garage.tf.models.mlp_merge_model import MLPMergeModel
+
+
+class CNNMLPMergeModel(Model):
+    """Convolutional neural network followed by a Multilayer Perceptron.
+
+    Combination of a CNN Model (optionally with max pooling) and an
+    MLP Merge model. The CNN accepts the state as an input, while
+    the MLP accepts the CNN's output and the action as inputs.
+
+    Args:
+        filter_dims (tuple[int]): Dimension of the filters. For example,
+            (3, 5) means there are two convolutional layers. The filter for
+            first layer is of dimension (3 x 3) and the second one is of
+            dimension (5 x 5).
+        num_filters (tuple[int]): Number of filters. For example, (3, 32) means
+            there are two convolutional layers. The filter for the first layer
+            has 3 channels and the second one with 32 channels.
+        strides (tuple[int]): The stride of the sliding window. For example,
+            (1, 2) means there are two convolutional layers. The stride of the
+            filter for first layer is 1 and that of the second layer is 2.
+        hidden_sizes (list[int]): Output dimension of dense layer(s).
```

Comment:
please make this tuple

Reply:
fixed

Reply:
It's usually better for the person who opened the conversation to resolve, also I still see it's `hidden_sizes (list[int])`

Reply:
Ah got it, yea I figured i'd fix all issues before pushing.